HW_19/task_5_new_class_using_type.py

```diff
@@ -0,0 +1,92 @@
+# Task 5. (optional) Create Bot and TelegramBot classes from
+# the first task using the type function
+
+def init_bot(self, name):
+    self.name = name
+
+
+def say_name(self):
+    print(self.name)
+
+
+def send_message(self, message):
+    print(f'{self.name}: {message}')
+
+
+Bot = type(
+    'Bot',
+    (),
+    {
+        '__init__': init_bot,
+        'say_name': say_name,
+        'send_message': send_message,
```

Comment:
Have fun with lambdas :)
```suggestion
        "__init__": lambda self, name: setattr(self, "name", name),
        "send_message": lambda self, message: print(message),
        "say_name": lambda self: print(self.name)
```